tests/defences/trainer/test_dp_instahide_trainer.py

```diff
@@ -24,6 +24,9 @@
 from art.defences.trainer import DPInstaHideTrainer
 from art.estimators.classification import PyTorchClassifier, TensorFlowV2Classifier, KerasClassifier
 from tests.utils import ARTTestException
+from tests.utils import get_image_classifier_hf
```

Comment:
Please remove the large model and integrate `get_image_classifier_hf` into `get_mnist_classifier`.

Reply:
Could we call `image_dl_estimator` instead?